### ISSUE TITLE
Finetune pretrained models

### ISSUE THREAD

**Issue by tczhangzhi** [FluxML/Metalhead.jl#60]
I'm trying to write a computer vision, graph convolutional neural network project to help others quickly use julia's deep learning template code.

I found that using the model provided by **Metalhead**, **finetune** can not achieve good results on the specified data set. Have you **frozen the relevant parameters**?

[This](https://github.com/tczhangzhi/Julia-Deeplearning/blob/master/VGG_and_Cifar.ipynb) is the code using a custom model, the loss can be reduced normally.

But when I replace it with the model in metalhead and want finetune, the loss is always above 2.30. I only replaced the model part with:

```
vgg = VGG19()
model = Chain(vgg.layers[1:end-6],
              Dense(512, 4096, relu),
              Dropout(0.5),
              Dense(4096, 4096, relu),
              Dropout(0.5),
              Dense(4096, 10)) |> device
Flux.trainmode!(model, true)
```

**Comment by tczhangzhi** [comment]
My bad, just reducing the lr works for me.
